File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/card/CardModel.java
package com.syahputrareno975.cardbattlemodule.model.card;

import java.io.Serializable;

public class CardModel implements Serializable {
    public String Id = "";
    public String Image = "";
    public String Name = "";
    public int Level  = 0;
    public Long Atk = 0L;
    public Long Def = 0L;
    public Long Price= 0L;
    public int Color = 0;

    public int Flag = 0;
    public String Message = "";


    public CardModel() {
    }

    public CardModel(String id, String image, String name, int level, Long atk, Long def, Long price, int color) {
        Id = id;
        Image = image;
        Name = name;
        Level = level;
        Atk = atk;
        Def = def;
        Price = price;
        Color = color;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/card/CardModel.kt
package com.syahputrareno975.simplecardbattle.model.card

import java.io.Serializable

class CardModel : Serializable {
    var Id = ""
    var Image = ""
    var Name = ""
    var Level  = 0
    var Atk: Long  = 0
    var Def: Long  = 0
    var Price: Long = 0
    var Color = 0

    var Flag = 0
    var Message = ""

    constructor()

    constructor(
        Id: String,
        Image: String,
        Name: String,
        Level: Int,
        Atk: Long,
        Def: Long,
        Price: Long,
        Color: Int
    ) {
        this.Id = Id
        this.Image = Image
        this.Name = Name
        this.Level = Level
        this.Atk = Atk
        this.Def = Def
        this.Price = Price
        this.Color = Color
    }


}